notebooks/resource_allocation/rec_cpu_req_lim.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>The change in <code>start_time</code> and <code>end_time</code> seems to have changed this graph, and the subsequent results and the markdown. Should those changes be reflected in this PR?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/operate-first-jupyterhub-analysis/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Good observation! The original `start_time` and `end_time` fetched data for a period when there was a significant data science workload on the Zero cluster. The analysis, results, and markdown in the notebook correspond to that time period. However that data is not available since we moved to Smaug so running the notebook for that dates would give an error. The purpose of this PR is to make sure the analysis run without any issues in the upcoming jupyter image for this work (issue #20). 
Two other relevant issues:
Issue #19 looks at how we can make the results and markdowns friendly for different starting conditions. 
Issue #26 looks at getting a snapshot of the original data.

Reply:
Cool, just wanted to make sure this is an expected change, and that issues have been made to address it :)